src/backend.rs

```diff
@@ -464,28 +226,215 @@ fn main() {}
+                }
+                Dependency::Detailed(odra_details)
+            }
+        }
+    }
+
+    fn project_dependency() -> Dependency {
+        Dependency::Detailed(DependencyDetail {
+            path: Some("..".to_string()),
+            features: vec!["wasm".to_string()],
+            optional: true,
+            default_features: Some(false),
+            ..Default::default()
+        })
+    }
+}
+
+/// Managing backends
+impl Backend {
+    /// Loads a backend from Odra.toml file
+    pub fn load(name: String) -> Backend {
+        let conf = OdraConf::load();
+        if conf.backends.is_none() {
+            Error::NoBackendConfigured.print_and_die();
+        } else {
+            let backends = conf.backends.unwrap();
+            let backend = backends.get(&name);
+            match backend {
+                None => {
+                    Error::NoSuchBackend.print_and_die();
+                }
+                Some(backend) => backend.clone(),
+            }
+        }
+    }
+
+    /// Prints out a table containing all backends
+    pub fn list() {
+        let backends = OdraConf::load().backends.unwrap_or_else(|| {
+            Error::NoBackendConfigured.print_and_die();
+        });
+
+        let mut table = Table::new();
+        table.set_header(vec!["Name", "Package", "Dependency"]);
+        for (_, backend) in backends {
+            table.add_row(vec![
+                backend.name,
+                backend.dependency_name,
+                toml::to_string(&backend.dependency).unwrap(),
+            ]);
+        }
+
+        println!("{}", table);
+    }
+
+    /// Removes backend from Odra.toml
+    pub fn remove(remove: RemoveBackendCommand) -> bool {
+        let mut conf = OdraConf::load();
+        if conf.backends.is_some() {
+            let mut backends = conf.backends.unwrap();
+            if backends.remove(&remove.name).is_some() {
+                conf.backends = Some(backends);
+                conf.save();
+                return true;
+            }
+        }
+
+        false
+    }
+
+    /// Adds backend to Odra.toml
+    pub fn add(add: AddBackendCommand) -> bool {
+        let mut conf = OdraConf::load();
+        let mut backends: HashMap<String, Backend>;
+
+        // If no name was passed, we use package
+        let name = match &add.name {
+            None => add.package.clone(),
+            Some(name) => name.clone(),
+        };
+
+        if conf.backends.is_none() {
+            backends = HashMap::new();
+        } else {
+            backends = conf.backends.unwrap();
+        }
+
+        if let Vacant(e) = backends.entry(name) {
+            e.insert(Backend::from_add_command(&add));
+            conf.backends = Some(backends);
+            conf.save();
+            true
+        } else {
+            false
+        }
+    }
+
+    fn from_add_command(add: &AddBackendCommand) -> Backend {
+        let dependency;
```

Comment:
You don't need to initialize `dependency` in every branch.
```suggestion
        let dependency = if ...
```